google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -250,6 +258,21 @@ class ConnectionWorker implements AutoCloseable {
   private static String projectMatching = "projects/[^/]+/";
   private static Pattern streamPatternProject = Pattern.compile(projectMatching);
 
+  private Meter writeMeter;
+  static AttributeKey<String> telemetryKeyWriterId = AttributeKey.stringKey("writer_id");
+  static List<AttributeKey<String>> telemetryKeysTraceId =
+      new ArrayList<AttributeKey<String>>() {
+        {
+          add(AttributeKey.stringKey("trace_field_0"));
```

Comment:
These names don't convey much meaning, this is some kind of parsing of the trace_id from the request?  Do we document this format anywhere?

Reply:
Dataflow lib_id:job-id:worker_id

Reply:
Yes, we extract the job name, job id and worker id fields for Dataflow.